<launch>
  <group ns="rtab">

    <!-- Visual SLAM -->
    <node name="rtabmap" pkg="rtabmap_ros" type="rtabmap" output="screen" args="">
      <param name="subscribe_depth" type="bool" value="true"/>
      <param name="subscribe_scan_cloud" type="bool" value="false"/>
      <param name="subscribe_stereo" type="bool" value="false"/>
      <param name="subscribe_scan" type="bool" value="false"/>

      <remap from="rgb/image"       to="/camera/left/image_rect_color"/>
      <remap from="rgb/camera_info" to="/camera/left/camera_info"/>
      <remap from="depth/image"     to="/camera/depth/image_raw"/>


      <remap from="odom" to="/odom"/>

      <remap from="proj_map" to="/map"/>
      <remap from="goal" to="/move_base_simple/goal"/>

      <param name="Mem/IncrementalMemory" type="string" value="false"/>

      <param name="frame_id" type="string" value="/base_link"/>
      <param name="map_frame_id" type="string" value="/map"/>
      <param name="queue_size" type="int" value="30"/>
      <param name="database_path" type="string" value="/media/ubuntu/361E-63D9/rtabmap.db" />
    </node>

  </group>
</launch>
